### ISSUE TITLE
Question about $_SESSION

### ISSUE THREAD

**Issue by MrNaitX** [fethica/PHP-Login#63]
This is not an issue, more of a question.
Where do put in the information in to $_SESSION? 
Can't seem to find it at all.

**Comment by therecluse26** [comment]
I'm sorry, could you rephrase? What are you trying to do specifically?

**Comment by MrNaitX** [comment]
Okey I will tell you my problem. When I login redirect back to my startpage. Then it takes 30-60 seconds or a couple of refreshes before $_SESSION has a value.
But it seems to be more timebased but can't be sure.
Like this:
if(!isset($_SESSION['username'])): ?>
        <div class="right"><span class="button menuButton dropdownTrigger" data-dropdown-id="1"><a>Logga In</a></div>
<?php else: ?>
        <div class="right"><span class="button menuButton"><a>Profil</a></div>
<?php endif; ?>

I already am doing a 301 for non-www. to www and I have session_start(); at the top of the .php.
So I'm trying to find where we put in the values in $_SESSION.

**Comment by MrNaitX** [comment]
So I have found it  but I can't really figure out why it updates so slow.

```
            if (password_verify($mypassword, $result['password']) && $result['verified'] == '1') {

                //Success! Register $myusername, $mypassword and return "true"
                $success = 'true';
                    session_start();

                    $_SESSION['username'] = $myusername;

            } elseif (password_verify($mypassword, $result['password']) && $result['verified'] == '0') {

                //Account not yet verified
                $success = "<div class=\"alert alert-danger alert-dismissable\"><button type=\"button\" class=\"close\" data-dismiss=\"alert\" aria-hidden=\"true\">&times;</button>Your account has been created, but you cannot log in until it has been verified</div>";

            } else {

                //Wrong username or password
                $success = "<div class=\"alert alert-danger alert-dismissable\"><button type=\"button\" class=\"close\" data-dismiss=\"alert\" aria-hidden=\"true\">&times;</button>Wrong Username or Password</div>";

            }
        }
        return $success;
```
